packages/typescript/src/services/completions/basic.ts

```diff
@@ -12,7 +12,12 @@ export interface Data {
 	uri: string,
 	fileName: string,
 	offset: number,
-	originalItem: ts.CompletionEntry;
+	originalItem: {
```

Comment:
Hm, but why don't `Pick<ts.CompletionEntry, '...'>`? Also didn't know some props can affect performance 🤔

Reply:
I found that with `Pick<ts.CompletionEntry, '...'>`, the following code doesn't report an error for missing `labelDetails`.

```
					originalItem: {
						name: tsEntry.name,
						source: tsEntry.source,
						data: tsEntry.data,
						// labelDetails: tsEntry.labelDetails,
					},
```